src/types.js

```diff
@@ -201,6 +201,12 @@ export type Outbox = {|
   subject: string,
   timestamp: number,
   type: 'stream' | 'private',
+
+  /**
+   * Whether the message is for local echo only. If true, the message is not
+   * sent to the server.
+   */
+  isLocalEcho?: boolean,
```

Comment:
Hmm, I think this needs a more precise name. The whole architecture for the "outbox" is kind of an implementation of the "local echo" concept; we've got a brief mention of "local echo" in a doc [here](https://github.com/zulip/zulip-mobile/blob/8bf48939771a4eacc8a170a4620ef2f01ff97d15/docs/glossary.md#local-echo).

At first, I thought, maybe, `isLocalEchoOnly`? But maybe that requires more effort than necessary to understand what it means.

I wonder about something like `deferSending`?

Reply:
`deferSending` sounds good. Changed.